### PR TITLE
Fix coreos_kernel_option oval check

### PR DESCRIPTION
**Pull request by JAORMX**
The regex was missing a `.`

### CODE REVIEW

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 757e53ad23f049d4dcd9a55a1965679702542c57

File: shared/templates/template_OVAL_coreos_kernel_option [hunk 1]
```diff
@@ -3,32 +3,32 @@
     <metadata>
       <title>Ensure that the most recent (default) CoreOS boot loader entry is configured to run Linux operating system with argument {{{ ARG_NAME_VALUE }}}</title>
       {{{- oval_affected(products) }}}
-      <description>Ensure {{{ ARG_NAME_VALUE }}} option is configured in the 'options' line in /boot/loader/entries/ostree-2-*.conf (or ostree-1-*.conf if the second version does not exists).</description>
+      <description>Ensure {{{ ARG_NAME_VALUE }}} option is configured in the 'options' line in /boot/loader/entries/ostree-2-.*.conf (or ostree-1-.*.conf if the second version does not exists).</description>
```
Comment (matejak) [inline shared/templates/template_OVAL_coreos_kernel_option:6]: Could you consider getting rid of this metadata boilerplate similarly to e.g. https://github.com/ComplianceAsCode/content/blob/master/shared/templates/template_OVAL_file_owner i.e. by using the `oval_metadata` macro? That would look like `{{{ oval_metadata("Ensure " + ARG_NAME_VALUE + " option is configured in the 'options' line in /boot/loader/entries/ostree-2-.*.conf (or ostree-1-.*.conf if the second version does not exist).") }}}`

**Comment by ggbecker** [comment]
@openscap-ci test this please

**Comment by evgenyz** [comment]
I'm reworking these rules and templates in #6100. I'll incorporate this fix. I'll also consider using `oval_metadata`

**Comment by JAORMX** [comment]
Abandoning in favor of https://github.com/ComplianceAsCode/content/pull/6100

**Pull request closed without merging**